src/index.html

```diff
@@ -1,13 +1,49 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <section class="card" data-qa="card">
+      <img
+         class="card__img"
+         src="./images/imac.jpeg"
+         alt="iMac"
+      >
+      <h2 class="card__title">Моноблок APPLE A1419 iMac<br> 27&quot; Retina 5K (MNED2UA/A)</h2>
+      <p class="card__code">Код товара: 195434</p>
+
+      <section class="stars">
+        <div class="stars__wrapper4">
```

Comment:
It is work, but, according to BEM, modificators should be written with a double line
```suggestion
        <div class="stars stars--4">
```